modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-webengine/src/main/resources/skin/views/distribution/forms.ftl

```diff
@@ -12,6 +12,7 @@
     <#assign rtSnap=Root.runtimeDistribution/>
     <#assign snapList=Root.listPersistedDistributions()/>
 
+  <div>
   <table class="tablesorter distributions">
```

Comment:
Indent issue

Reply:
I'm not always reindenting on HTML otherwise it makes the diff unreadable.
The status i inherited in this module is very bad, so if we have guideline and a way to reformat automatically, coud you please point me to it?

Reply:
(anyway i'll remove the div)

Reply:
https://github.blog/2018-05-01-ignore-white-space-in-code-review/

I don't expect a full format; as you said it will change the whole file. But when adding a container tag; it makes sense to adjust the content indent. (using `tab` key, or most of recent IDE has this feature by default. For instance: https://mspoweruser.com/visual-studio-code-can-now-indent-code-automagically/)